docs/core/compatibility/syslib-warnings/syslib0013.md

```diff
@@ -0,0 +1,17 @@
+---
+title: SYSLIB0013 warning
```

Comment:
`https://aka.ms/dotnet-warnings/SYSLIB0013` may need to get updated to this article after it's published. (Same applies to other obsoletion warnings.

Reply:
@jeffhandley I don't seem to have access to these aka.ms links. Can you update for 13-16, 19-20?

Reply:
I will publish these live now.

Reply:
I went ahead and created the aka.ms links for SYSLIB0017 through SYSLIB0025. I don't (yet) have access to edit the existing ones, but I submitted a request to gain access.